.env.example

```diff
@@ -3,7 +3,7 @@ DB_HOST="127.0.0.1"
 DB_PORT="3306"
 DB_NAME="final_project_db"
 DB_USER="root"
-DB_PASSWORD="root"
+DB_PASSWORD="Password@1"
```

Comment:
You're not supposed to edit the `.env.example` file. It's just a convenience so it is easy for you to make a `.env` file. The `.env` file is gitignored so it won't show up in git. The `.env.example` is not.

I guess you just revealed your database password to the world 😱

Reply:
Oh  I will revert the changes I made to .env.example.Ok now I understood the mistake